src/main/resources/db/migration/V1_12__object_directory_schema_data.sql

```diff
@@ -33,7 +33,7 @@ CREATE TABLE IF NOT EXISTS external_object_directory (
 , external_location_type        CHARACTER VARYING           NOT NULL
 , created_ts                    TIMESTAMP WITH TIME ZONE    NOT NULL
 , modified_ts                   TIMESTAMP WITH TIME ZONE    NOT NULL
-, modified_by                   INTEGER                     NOT NULL
+, modified_by                   INTEGER                     --NOT NULL
```

Comment:
Intentional to leave this commented?

Reply:
Yes, I'm afraid it has to be commented out at the moment because we don't have `moj_user` currently in our flyway scripts